.circleci/config.yml

```diff
@@ -308,83 +300,66 @@ jobs:
 workflows:
   CI:
     jobs:
-      - checks:
-          filters: # required since publish jobs have tag filters and dependencies
-            tags:
-              only: /.*/
```

Comment:
As discussed during stand-up, we won't run the tests when tagging. That's the default behavior in CircleCI, hence the removal of this config settings.